common/include/common/float_comparison.hpp

```diff
@@ -32,7 +32,7 @@
 #include <complex>
 #endif
 
-#ifdef BLAS_DATA_TYPE_HALF
+#ifdef BLAS_ENABLE_HALF
```

Comment:
I think that all the `BLAS_ENABLE_HALF` guards in this file can be safely removed.

Reply:
Agree, there were some that were already there. I guess we can remove them as well.

Reply:
Whoopsie! Thanks!